### PR TITLE
feat: includes not annotated logos in search

### LINKED ISSUE #262: Don’t only show annotated logos in /logos/search
In https://hunger.openfoodfacts.org/logos/search/, we only show annotated logos. However, this page is really useful to display all logos of a specific product (annotated or not). This way, we can boostrap annotation of logos that are not annotated yet, by selecting products that we know have this logo.

### PR DESCRIPTION
**Pull request by alexfauquette**
### Fixes bug(s)
- Fix #262

image

See the issue comment. I'm not sure about how the API responds when `annotated` parameter is not provided

### CODE REVIEW

File: src/robotoff.ts [hunk 1]
```diff
@@ -90,7 +90,7 @@ const robotoff = {
   searchLogos(barcode, value, type, count = 25) {
     return axios.get(`${ROBOTOFF_API_URL}/images/logos`, {
```
Comment (raphael0202) [inline src/robotoff.ts:91]:
```suggestion
    return axios.get(`${ROBOTOFF_API_URL}/images/logos/search`, {
```

**Review by raphael0202 — CHANGES_REQUESTED** [review]
see suggestion about new search route

**Comment by alexfauquette** [comment]
Should be merged after robotof deployment of the search route. For now, this returns 404.

https://robotoff.openfoodfacts.org/api/v1/images/logos/search/?barcode=3560071212810&count=25

**Comment by raphael0202** [comment]
It's now merged but not deployed. I plan to deploy next Monday.

**Comment by alexfauquette** [comment]
Nice, if you want you can merge it after deploy is done 👍

Merging on master triggers the deploy github action

**Comment by raphael0202** [comment]
Merging this as the Robotoff update has been deployed to production

**Review by raphael0202 — APPROVED** [review]

**Pull request merged**